valhalla/odin/maneuver.h

```diff
@@ -378,6 +381,7 @@ class Maneuver {
   uint32_t begin_heading_;
   uint32_t end_heading_;
   uint32_t roundabout_exit_begin_heading_;
```

Comment:
Need to move roundabout_exit_begin_heading_ and roundabout_exit_turn_degree_ down with roundabout_exit_length_. The same applies for all the methods and print outs

Reply:
Sure, I can move all of the roundabout_exit_*'s since roundabout_exit_begin_heading_ and a few others were added in the wrong location.